ci/common.sh

```diff
@@ -75,6 +75,15 @@ install_curl () {
     curl -V
 }
 
+install_apisix_runtime() {
+    export runtime_version=${APISIX_RUNTIME}
+    # I need to use script from my repo to test
```

Comment:
?

Reply:
I need to use feature branch to pass the CI, So I use link which comes from my own repo to test. After finishing review, I will rever this change
ref: https://github.com/api7/apisix-build-tools/pull/367